config/routes.rb

```diff
@@ -29,13 +29,14 @@
       resources :participants, only: [:index]
     end
   end
-  resources :sponsors, only: [:index]
-  resources :series, only: [:show]
-  resources :team_members, path: "team", only: [:index, :show]
 
   put "newsletter_signup", to: "newsletter_signups#create"
   match "*path/newsletter_signup", to: "newsletter_signups#create", via: :put
 
+  resources :sponsors, only: [:index]
+  resources :series, only: [:show]
+  resources :team_members, path: "team", only: [:index, :show]
```

Comment:
Did you mean to move these?

Reply:
I realized I added newsletter_signup out of alphabetical order and figured this PR would be a good place to move it back into position.